package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "description": "Create salary bands for your company in an anonymous and safe way.",
   "author": "Sebastian Di Luzio",
-  "packageManager": "pnpm@9.15.9",
+  "packageManager": "pnpm@10.10.0",
```

Comment:
I think the only breaking change that affects us is the lifecycle change, as at least sqlite needs to run postinstalls

https://github.com/pnpm/pnpm/blob/HEAD/pnpm/CHANGELOG.md#major-changes